AirCasting/SDSync/ViewsFlow/SDSyncSuccess/SDSyncSuccessView.swift

```diff
@@ -0,0 +1,69 @@
+// Created by Lunar on 01/12/2021.
+//
+
+import AirCastingStyling
+import SwiftUI
+
+struct SDSyncSuccessView<VM: SDSyncSuccessViewModel>: View {
```

Comment:
It the view which shows that the whole SDSync was successful or that the backend sync was completed?